LPS-64215 Uses "lfr-icon-menu" like in the non lexicon view

ealonso opened the pull request.

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest(master)/1141/">test-portal-acceptance-pullrequest(master)</a></p>

ealonso commented:
Hey @evannagayama,

I've replaced the old icons for the lexicon icons, I've replaced plus icon and ellipsis icon on portlet topper.

Could you take a look at poshi errors?

Thanks!!!!

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 44 minutes 37 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/086e016380206694a752086df5a08b7534fb81ee">086e016380206694a752086df5a08b7534fb81ee</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest(master)/1141/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28042/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24360/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><strike><strong><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28043/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24361/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25692/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23893/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/36200/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><strike><strong><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21402/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></strong></strike></li><li><strike><strong><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32452/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></strong></strike></li><li><strike><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23476/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></strong></strike></li><li><strike><strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/36201/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></strong></strike></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23894/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23732/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21403/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><strike><strong><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25302/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></strong></strike></li><li><strike><strong><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25693/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></strong></strike></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32276/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23895/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24742/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23733/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/24</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25694/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/25</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32853/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/26</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32277/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/27</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31493/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><strike><strong><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27733/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></strong></strike></li><li><strike><strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24714/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></strong></strike></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/36199/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23892/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21401/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23475/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24743/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23734/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32278/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32854/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25695/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24744/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21404/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32153/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23896/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28044/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31494/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28045/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk7</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23735/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23897/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25303/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/1120/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1141/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest(master)/1141/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>37 Jobs Passed.<br />10 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28043/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></h5><h6>Job Results:</h6><p>8 Tests Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/28043/">test-portal-acceptance-pullrequest-batch&#40;master&#41; ?? 3,!master #28043</a></strong><pre>
     \[echo\] Merging into file /opt/dev/projects/github/liferay\-jenkins\-ee/test\-results/test\-suite/TESTS\-test\-portal\-acceptance\-pullrequest\(master\)\_1141\_functional\-tomcat8\-mysql56\-jdk7\_3\.xml\.
     \[echo\] Merging /opt/dev/projects/github/liferay\-jenkins\-ee/test\-results/TEST\-backup\-results\.xml\.
     \[echo\] Sending testray results\.
     \[exec\] sending incremental file list
     \[exec\] TESTS\-test\-portal\-acceptance\-pullrequest\(master\)\_1141\_functional\-tomcat8\-mysql56\-jdk7\_3\.xml
     \[exec\] 
     \[exec\] sent 1,096 bytes  received 35 bytes  2,262\.00 bytes/sec
     \[exec\] total size is 3,000  speedup is 2\.65
   \[delete\] Deleting: /opt/dev/projects/github/liferay\-jenkins\-ee/commands/20160314103449554
   \[delete\] Deleting: /opt/dev/projects/github/liferay\-jenkins\-ee/testray\.build\.properties
   \[delete\] Deleting: /opt/dev/projects/github/liferay\-jenkins\-ee/testray\.testcases\.properties
     \[copy\] Copying 1 file to /mnt/mfs\-ssd1\-10\.0\.10/jenkins/test\-1\-17/opt/java/jenkins/userContent/jobs/test\-portal\-acceptance\-pullrequest\(master\)/builds/1141/testray\-results
     \[echo\] Finished recording Testray results\.

BUILD FAILED
/opt/dev/projects/github/liferay\-jenkins\-ee/commands/build\-test\-portal\-acceptance\-pullrequest\-batch\.xml:332: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:782: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:538: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:586: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-dist\.xml:605: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\.xml:284: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-common\-java\.xml:233: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-common\-java\.xml:114: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-common\-java\.xml:133: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/util\-bridges/build\.xml:28: The following error occurred while executing this line:
/opt/dev/projects/github/liferay\-portal/build\-common\-java\.xml:172: Failed to build jar file:

</pre></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/21402/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></h5><h6>Job Results:</h6><p>1 Test Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/21402//testReport/com.liferay.poshi.runner/PoshiRunner/test_WebcontentpublicationUsecase_AssetPublisherSiteAndGlobalScopeAccessibility_">PoshiRunner.test[WebcontentpublicationUsecase#AssetPublisherSiteAndGlobalScopeAccessibility]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/functional-tomcat8-mysql56-jdk7/13">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/functional-tomcat8-mysql56-jdk7/13">Poshi Summary</a> - <a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/21402//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32452/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></h5><h6>Job Results:</h6><p>23 Tests Passed.<br />3 Tests Failed.</p><ol><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/32452//testReport/com.liferay.poshi.runner/PoshiRunner/test_ApplicationdisplaytemplatesUsecase_ADTBreadcrumb_">PoshiRunner.test[ApplicationdisplaytemplatesUsecase#ADTBreadcrumb]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Summary</a> - <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/32452//console">Console Output</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/32452//testReport/com.liferay.poshi.runner/PoshiRunner/test_ApplicationdisplaytemplatesUsecase_AddRSSADTViaPortletConfiguration_">PoshiRunner.test[ApplicationdisplaytemplatesUsecase#AddRSSADTViaPortletConfiguration]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Summary</a> - <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/32452//console">Console Output</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/32452//testReport/com.liferay.poshi.runner/PoshiRunner/test_Calendar_ViewWeekStartMonday_">PoshiRunner.test[Calendar#ViewWeekStartMonday]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Summary</a> - <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/32452//console">Console Output</a></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23476/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></h5><h6>Job Results:</h6><p>17 Tests Passed.<br />9 Tests Failed.</p><ol><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGBookmarks_AddBookmark_">PoshiRunner.test[PGBookmarks#AddBookmark]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Summary</a> - <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//console">Console Output</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGBookmarks_AddFolder_">PoshiRunner.test[PGBookmarks#AddFolder]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Summary</a> - <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//console">Console Output</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGBookmarks_AddFolderBookmark_">PoshiRunner.test[PGBookmarks#AddFolderBookmark]</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Report</a> - <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/1141/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/functional-tomcat8-mysql56-jdk7/15">Poshi Summary</a> - <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/23476//console">Console Output</a></li><li>...</li></ol><p><strong>Click <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23476//testReport/">here</a> for more failures.</strong></p></li><li>...</li></ol>

evannagayama commented:
@ealonso there is a bug from this pr that the options menu expands outside of the page for the search portlet (and i'm assuming any portlet that is too close to the right side). Could you fix this and resend? Thanks!

![offsetsearchoptions](https://cloud.githubusercontent.com/assets/2288514/13795797/3e39c152-eac0-11e5-9ad3-7bd08d482eee.png)

ealonso commented:
see: https://github.com/evannagayama/liferay-portal/pull/648
